phoenicis-tools/src/main/java/org/phoenicis/tools/ToolsConfiguration.java

```diff
@@ -153,6 +153,6 @@ public Opener opener() {
 
     @Bean
     public LnkParser lnkParser() {
-        return new LnkParser();
+        return new LnkParser(new org.phoenicis.lnk.LnkParser());
```

Comment:
Can we rename the `LnkParser` class in Phoenicis so that we don't have a naming conflict?

Reply:
Any better idea of name?

Reply:
`PhoenicisLnkParser`? It is basically a wrapper class for the moved `LnkParser` right? So the name is not really important as long as it retains its functional meaning.

Reply:
This will be used in the script, so the name is important

Reply:
We could also name it `LnkParserWrapper`, depending on whether the class name is exposed to the scripts...

Reply:
Hmmm, in that case it is hard to find a different name, I agree.